configs/interaction/environment/unity.yaml

```
@@ -1,6 +1,7 @@
 _target_: ami.interactions.environments.unity_environment.UnityEnvironment
 
 file_path: ???
+log_file_path: ???
```

デフォルト値`None`でお願いします。
```suggestion
log_file_path: null
```

---

https://github.com/MLShukai/ami-vconf24/pull/55/commits/b359ff15036cd281c15d3e2286fc2098b8f9a035
修正しました